openedx/core/djangoapps/oauth_dispatch/dot_overrides.py

```diff
@@ -99,7 +101,10 @@ def save_bearer_token(self, token, request, *args, **kwargs):
             assert expires_in < 0
 
             token['expires_in'] = expires_in
-
+        log.info(
```

Comment:
`log.info('Finished bearer token with prefix [%s] for user [%d]', _token_prefix, user.id)`